gtfu/src/main/java/gtfu/tools/Slack.java

```diff
@@ -0,0 +1,32 @@
+package gtfu.tools;
+import gtfu.HTTPClient;
+import gtfu.Debug;
+
+/**
+ * Interact with Slack
+ * @see <a href="https://slack.dev/java-slack-sdk/guides/incoming-webhooks</a>
+ */
+public class Slack {
+    private String webHookURL;
+
+    /**
+     * Initialize Slack connection
+     */
+    public Slack() {
+        this.webHookURL = System.getenv("GRAAS_BOT_WEBHOOK");
+        Debug.error("* No env variable found with name GRAAS_BOT_WEBHOOK...exiting.");
```

Comment:
Something happened here. The null check of webHookURL disappeared. Also a reminder to not call System.exit() from code that may be called from other code, as it may lead to very unpleasant debugging down the road.

Reply:
oof!!